includes/filerepo/file/LocalFile.php

```diff
@@ -1062,7 +1061,6 @@ function recordUpload2(
 				#throw new MWException( "Empty oi_archive_name. Database and storage out of sync?" );
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS125)